packages/angular_devkit/build_angular/package.json

```diff
@@ -51,7 +51,7 @@
     "webpack-dev-server": "3.1.10",
     "webpack-merge": "4.1.4",
     "webpack-sources": "1.3.0",
-    "webpack-subresource-integrity": "1.1.0-rc.6"
+    "webpack-subresource-integrity": "1.2.0"
```

Comment:
Leave this at the existing version.  1.2.0 has an unnecessary peer dependency on `html-webpack-plugin` which will cause a peer dependency warning for every CLI user.

Reply:
@clydin updated.